Library/Homebrew/dependency_collector.rb

```diff
@@ -117,6 +117,8 @@ def parse_symbol_spec(spec, tags)
     when :emacs      then EmacsRequirement.new(tags)
     # Tiger's ld is too old to properly link some software
     when :ld64       then LD64Dependency.new if MacOS.version < :leopard
+    # Tiger doesn't ship expat in /usr/lib
+    when :expat        then Dependency.new("expat", tag) if MacOS.version < :leopard
```

Comment:
Feel free to throw something at me for being this anal but the `then` seems too indented here.